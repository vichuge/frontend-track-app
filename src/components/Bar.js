import React from 'react';
import PropTypes from 'prop-types';

const Bar = ({ title }) => (
  <nav className="navbar">
    <h1 className="title">{title}</h1>
  </nav>
);

Bar.propTypes = {
  title: PropTypes.string.isRequired,
};

export default Bar;
