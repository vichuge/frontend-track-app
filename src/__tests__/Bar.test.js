import React from 'react';
import renderer from 'react-test-renderer';
import Bar from '../components/Bar';

describe('Bar', () => {
  it('return Bar component', () => {
    const tree = renderer
      .create(<Bar title="testTitle" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
