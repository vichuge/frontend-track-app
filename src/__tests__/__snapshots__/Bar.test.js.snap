// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bar return Bar component 1`] = `
<nav
  className="navbar"
>
  <h1
    className="title"
  >
    <i
      className="fas fa-tint"
    />
    Â 
    testTitle
  </h1>
</nav>
`;
